Resources/ServerInfo/Guidebook/_CD/sop/core.xml

```diff
@@ -0,0 +1,61 @@
+<Document>
+  # GENERAL STANDARD OPERATING PROCEDURES
+
+  The standard procedures followed on any uniform NT station. The Captain can edit and make additions to their station's general operation procedures (excluding Contraband) whenever they please as long as a copy of the changes is forwarded to each station bound fax machine. Anyone in the station's sector is required to follow this document and their related work SOP.
```

Comment:
Faxing a copy to every single fax would be a massive pain.

Reply:
I don't know how else to be certain it's properly relayed for new-arrivals.

Reply:
Yeah, I really can't think of a better solution ether.